client/reader/site-subscription/site-subscription.tsx

```diff
@@ -16,9 +16,13 @@ const SiteSubscription = ( { blogId, subscriptionId, transition }: SiteSubscript
 	return (
 		<SubscriptionManagerContextProvider portal={ SubscriptionsPortal.Reader }>
 			<SiteSubscriptionProvider blogId={ blogId } subscriptionId={ subscriptionId }>
-				<Main className="site-subscriptions-manager">
-					<ReaderSiteSubscription transition={ transition } />
-				</Main>
+				<div>
+					<div>
+						<Main className="site-subscriptions-manager">
+							<ReaderSiteSubscription transition={ transition } />
+						</Main>
+					</div>
+				</div>
```

Comment:
This is really confusing. Have no idea how it actually works...

Reply:
@enejb can you elaborate why above structure was needed?